client/js/table-model.js

```diff
@@ -16,6 +16,25 @@ class TableModel {
   setValue(location, value) {
     this.data[this._getCellId(location)] = value;
   }
-}
 
+  
+  _getColNums(col) {
+    let nums = [];
+    for ( row = 0; row < this.numRows; row++) {
```

Comment:
"row" should have an identifier ("let") so that it doesn't become a global variable.